src/prisma/migrations/20240704112555_ip_allowlist/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "configuration" ADD COLUMN     "ipAllowlist" TEXT[] DEFAULT ARRAY[]::TEXT[];
```

Comment:
Why `TEXT[]` and not `String[]` or just `String`? 

Maybe we can make it `nullable` instead of `empty`